src/views/MetocDataView.vue

```diff
@@ -758,6 +794,20 @@ export default class MetocDataView extends Mixins(WMSMixin, TimeSeriesMixin) {
     return ['==', 1, 0]
   }
 
+  locationIsInFeatures(): boolean {
+    const source = this.locationsLayerOptions.source as GeoJSONSourceOptions
```

Comment:
Moeten er nog `null`/`undefined` checks bij op `source` en `source.data`? Of zijn die gegarandeerd gezet?